specification/cognitiveservices/data-plane/HealthInsights/preview/2023-03-01-preview/openapi.json

```diff
@@ -78,7 +78,8 @@
             "in": "body",
             "schema": {
               "$ref": "#/definitions/OncoPhenotypeData"
-            }
+            },
+            "required": true
```

Comment:
@markcowl: What is causing the addition of "required" here?  Is this a breaking change?